src/server/loginHandler.js

```diff
@@ -0,0 +1,35 @@
+const db = require('./loginDatabase');
+
+function login(req, res) {
+  res.setHeader('Content-Type', 'application/json');
+  let email = req.body.email;
+  let password = req.body.password;
+  let status = 200;
+  let contentType = req.headers['content-type'].toLowerCase();
+  let obj = {};
+  if (email === '' || password === '') {
+    obj = {'errorType': 'FieldMissing'};
+    status = 400;
+  } else if (contentType !== 'application/json') {
+    obj = {'errorType': 'ContentType'};
+    status = 400;
+  }
+
+  if (status !== 200) {
+    res.status(status).json(obj);
+  } else {
+    let result = db.checkUserInfo(email, password);
```

Comment:
this is not working, mongo is asynchronous. You have to pass a callback to this function and send the response from that callback.